polaris-tokens/src/tokens.ts

```diff
@@ -65,10 +65,10 @@ export interface Tokens {
 }
 
 export const tokens: Tokens = {
-  breakpoints: tokensToRems(breakpoints),
+  legacyTokens: tokensToRems(legacyTokens),
+  breakpoints,
```

Comment:
We converted these tokens to `rem` for use in components. e.g. Layout components that want to set `max-width`s at a given breakpoint.

Would it be better to update the logic in `gen-media-queries`?